overlays/DIMO/aftermarket.vspec

```diff
@@ -0,0 +1,96 @@
+# cell provides fields related to dimo device's cellular signal.
+# Dependencies to other overlays: None
+# Known conflicts with other overlays: None
+#
+HDOP:
+  datatype: float
+  type: sensor
+  description: Horizontal dilution of precision of GPS
+
+NSAT:
+  datatype: float
+  type: sensor
+  description: Number of sync satellites for GPS
+
+WPAState:
+  datatype: string
+  type: sensor
+  description: Indicate the current WPA state for the device's wifi
```

Comment:
I'd be interested in listing the possible values for this.

Reply:
e.g. "CONNECTED", "SCANNING", "DISCONNECTED"